python/servo/build_commands.py

```diff
@@ -257,14 +261,15 @@ def build(self, target=None, release=False, dev=False, jobs=None,
             shutil.copy(path.join(self.android_support_dir(), "openssl.makefile"), openssl_dir)
             shutil.copy(path.join(self.android_support_dir(), "openssl.sh"), openssl_dir)
             env["ANDROID_NDK_ROOT"] = env["ANDROID_NDK"]
+            env["RUST_TARGET"] = target
             with cd(openssl_dir):
                 status = call(
                     make_cmd + ["-f", "openssl.makefile"],
                     env=env,
                     verbose=verbose)
                 if status:
                     return status
-            openssl_dir = path.join(openssl_dir, "openssl-1.0.1t")
+            openssl_dir = path.join(openssl_dir, "openssl-1.1.0f")
```

Comment:
Ok, it makes sense to move the OpenSSL update into a separate PR. The reason for the update is that OpenSSL 1.1.0 comes with a compilation target ready for arm64, while 1.0.1 required a lot of hacking (and it's a out of support version)